### PR TITLE
Create prefilled issue template prototype

### PR DESCRIPTION
**Pull request by Quezler**
> experiment to fill the fields of the issue template automatically to make reporting bugs less repetitive copy pasting

keeping this in draft mode for several reasons:
- some `java.` packages were used
- its a hardcoded copy of the issue template in `.github`
- code is a bit unreadable
- anuke might have done (or wanted it) differently
- etc

(and yes the color is plastanium) 

Screen Shot 2020-11-20 at 11 54 43
Screen Shot 2020-11-20 at 11 54 29

[`https://github.com/Anuken/Mindustry/issues/new?assignees=&labels=bug&body=**Platform**%3A+%60Mac+OS+X%60%0A%0A**Build**%3A+%60bleeding-edge+build+11414%60%0A%0A**Issue**%3A+*Explain+your+issue+in+detail.*%0A%0A**Steps+to+reproduce**%3A+*How+you+happened+across+the+issue%2C+and+what+exactly+you+did+to+make+the+bug+happen.*%0A%0A**Link%28s%29+to+mod%28s%29+used**%3A+%60%5BDeltaNedas%2Fdev-mode%3A1.3.0%5D%60%0A%0A**Save+file**%3A+*The+%28zipped%29+save+file+you+were+playing+on+when+the+bug+happened.+THIS+IS+REQUIRED+FOR+ANY+ISSUE+HAPPENING+IN-GAME%2C+REGARDLESS+OF+WHETHER+YOU+THINK+IT+HAPPENS+EVERYWHERE.+DO+NOT+DELETE+OR+OMIT+THIS+LINE+UNLESS+YOU+ARE+SURE+THAT+THE+ISSUE+DOES+NOT+HAPPEN+IN-GAME.*%0A%0A**Crash+report**%3A+*The+contents+of+relevant+crash+report+files.+REQUIRED+if+you+are+reporting+a+crash.*%0A%0A---%0A%0A*Place+an+X+%28no+spaces%29+between+the+brackets+to+confirm+that+you+have+read+the+line+below.*%0A-+%5B+%5D+**I+have+searched+the+closed+and+open+issues+to+make+sure+that+this+problem+has+not+already+been+reported.**`](https://github.com/Anuken/Mindustry/issues/new?assignees=&labels=bug&body=**Platform**%3A+%60Mac+OS+X%60%0A%0A**Build**%3A+%60bleeding-edge+build+11414%60%0A%0A**Issue**%3A+*Explain+your+issue+in+detail.*%0A%0A**Steps+to+reproduce**%3A+*How+you+happened+across+the+issue%2C+and+what+exactly+you+did+to+make+the+bug+happen.*%0A%0A**Link%28s%29+to+mod%28s%29+used**%3A+%60%5BDeltaNedas%2Fdev-mode%3A1.3.0%5D%60%0A%0A**Save+file**%3A+*The+%28zipped%29+save+file+you+were+playing+on+when+the+bug+happened.+THIS+IS+REQUIRED+FOR+ANY+ISSUE+HAPPENING+IN-GAME%2C+REGARDLESS+OF+WHETHER+YOU+THINK+IT+HAPPENS+EVERYWHERE.+DO+NOT+DELETE+OR+OMIT+THIS+LINE+UNLESS+YOU+ARE+SURE+THAT+THE+ISSUE+DOES+NOT+HAPPEN+IN-GAME.*%0A%0A**Crash+report**%3A+*The+contents+of+relevant+crash+report+files.+REQUIRED+if+you+are+reporting+a+crash.*%0A%0A---%0A%0A*Place+an+X+%28no+spaces%29+between+the+brackets+to+confirm+that+you+have+read+the+line+below.*%0A-+%5B+%5D+**I+have+searched+the+closed+and+open+issues+to+make+sure+that+this+problem+has+not+already+been+reported.**)

### CODE REVIEW

**Comment by summetdev** [comment]
> its a hardcoded copy of the issue template in .github

So maybe just fetch bug_report.md file?

**Comment by Quezler** [comment]
@summetdev well it can already receive a copy during compilation, but for this prototype didn't want to bother injecting the values in that file since it would be a moderate puzzle to get right

File: core/src/mindustry/ui/Links.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public LinkEntry(String name, String link, Drawable icon, Color color){
...
+            return base + "&body=" + URLEncoder.encode(Strings.format("**Platform**: `@`\n\n**Build**: `@`\n\n**Issue**: *Explain your issue in detail.*\n\n**Steps to reproduce**: *How you happened across the issue, and what exactly you did to make the bug happen.*\n\n**Link(s) to mod(s) used**: `@`\n\n**Save file**: *The (zipped) save file you were playing on when the bug happened. THIS IS REQUIRED FOR ANY ISSUE HAPPENING IN-GAME, REGARDLESS OF WHETHER YOU THINK IT HAPPENS EVERYWHERE. DO NOT DELETE OR OMIT THIS LINE UNLESS YOU ARE SURE THAT THE ISSUE DOES NOT HAPPEN IN-GAME.*\n\n**Crash report**: *The contents of relevant crash report files. REQUIRED if you are reporting a crash.*\n\n---\n\n*Place an X (no spaces) between the brackets to confirm that you have read the line below.*  \n- [ ] **I have searched the closed and open issues to make sure that this problem has not already been reported.**", System.getProperty("os.name"), Version.combined(), Vars.mods.list().select(LoadedMod::enabled).map(l -> l.meta.author + "/" + l.name + ":" + l.meta.version)), StandardCharsets.UTF_8.toString());
```
Comment (Anuken) [inline core/src/mindustry/ui/Links.java:63]: Use `Strings.encode`, which is compatible with Android and doesn't throw an exception.

Comment (Quezler) [reply core/src/mindustry/ui/Links.java]: changed, now its just one line, i might as well just add it inline as a link, but keeping it in a function would probably be better if in future it maybe reads the issue template file and overrides some variables, do let me know :) 🍿

**Comment by Anuken** [comment]
See linked commit.

**Pull request closed without merging**